.packit.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# Packit's default fix-spec-file often doesn't fetch version string correctly.
+# This script handles any custom processing of the dist-git spec file and gets used by the
+# fix-spec-file action in .packit.yaml
+
+set -eo pipefail
+
+# Get Version from HEAD
+VERSION=$(grep '^version' Cargo.toml | cut -d\" -f2 | sed -e 's/-/~/')
```

Comment:
At your next opportunity, I would suggest also adding a comment to the `Cargo.toml` file pointing back here.  This might avoid somebody inadvertently breaking this script by format changes in `Cargo.toml`.

Reply:
done